parameters.d-prerelease/osg36-el8.yaml

```
@@ -0,0 +1,16 @@
+platforms:
+  - centos_8_x86_64
+
+sources:
+  - opensciencegrid:master; 3.6; osg-prerelease
+  - opensciencegrid:master; 3.6; osg > osg-prerelease
+  - opensciencegrid:master; 3.6; osg-prerelease, osg-upcoming-prerelease, osg-upcoming
+  - opensciencegrid:master; 3.6; osg > osg-prerelease, osg-upcoming-prerelease, osg-upcoming
```

```suggestion
  # - opensciencegrid:master; 3.6; osg-prerelease, osg-upcoming-prerelease, osg-upcoming
  # - opensciencegrid:master; 3.6; osg > osg-prerelease, osg-upcoming-prerelease, osg-upcoming
```

We don't need to run these until we have packages in 3.6-upcoming